.gitignore

```diff
@@ -1,22 +1,34 @@
-logs
-project/project
-project/target
-target
-lib_managed
-tmp
-.history
-dist
-/.idea
+*.iws
+*.pyc
+*.tgz
+.idea/
+.pytest_cache/
+.vscode/
 /*.iml
 /*.ipr
-/out
-/.idea_modules
 /.classpath
+/.coverage
+/.envrc
+/.history
+/.idea_modules
 /.project
-/RUNNING_PID
 /.settings
-*.iws
-node_modules/
-npm-debug.log
-yarn-debug.log
-yarn-error.log
\ No newline at end of file
+/.version
+/RUNNING_PID
+/build
+/out
+__pycache__
+coverage.xml
+dist
+canary-packages
```

Comment:
Not alphabetical?

Reply:
This has been fixed with a cruft update 👍